app/src/main/java/org/eyeseetea/malariacare/domain/exception/NullContextException.java

```diff
@@ -0,0 +1,17 @@
+package org.eyeseetea.malariacare.domain.exception;
+
+
+public class NullContextException extends Exception {
```

Comment:
We should create domain exception when the problem is an expected and functional exception but if context is null is a developer error, I think that we don´t need a NullContextException.